### PR TITLE
Fix a visual bug

### PR DESCRIPTION
**Pull request by NielsPilgaard**
## Description

I found a small error, so I thought I'd fix it 😊
The previous indentation causes the codeblock to not render on GitHub.

## Quality

- [x] Documentation is updated
- [ ] Release notes are updated
- [ ] Package version is updated
- [ ] Public types and methods are documented
- [ ] Tests are implemented and executed locally

### CODE REVIEW

File: source/Logging/documents/documentation.md [hunk 1]
```diff
@@ -6,16 +6,24 @@ Request and response stream should be seekable for the middleware to read stream
...
+
 ---
 
-    serviceCollection.AddScoped<IRequestResponseLogging>(
+``` c#
+serviceCollection.AddScoped<IRequestResponseLogging>(
     _ => new RequestResponseLoggingBlobStorage(connectionString,
                                                 containerName,
                                                 ILogger<RequestResponseLoggingBlobStorage>));
+````
```
Comment (NielsPilgaard) [inline source/Logging/documents/documentation.md:26]:
```suggestion
```
```

Comment (NielsPilgaard) [inline source/Logging/documents/documentation.md:26]:
```suggestion
```
```

**Comment by NielsPilgaard** [comment]
We have a bit too many backticks now 😄

**Comment by Sondergaard** [comment]
Hi @NielsPilgaard,

thank you for your contribution. 

We have hit a minor bump with our pipelines, as they are not ready for contributions from a forked repository.

If you don't mind, then I will move your work into an internal branch and get that merged into the codebase. I will keep a link to this PR and mention of your GitHub handle.

**Pull request closed without merging**